app/models/tests/test_test_group_model.py

```diff
@@ -50,13 +50,10 @@ def test_group_doc_to_dict(self):
         test_group.cross_compile = "cross-compile"
```

Comment:
@khilman The fields being removed below don't appear to have ever been used anywhere in production.  Was it used on an other instance that you know of?